fe1-web/src/App.tsx

```diff
@@ -29,6 +30,42 @@ configureNetwork(messageRegistry, keyPairRegistry);
 // start persisting the redux state after all reducers have been registered
 persist.persist();
 
+const BuildInfo = () => {
+  const styles = StyleSheet.create({
+    container: {
+      position: 'absolute',
+      bottom: '4px',
+      left: '3px',
```

Comment:
If possible it would be great if the "standard" spacing units (https://github.com/dedis/popstellar/blob/master/fe1-web/src/core/styles/spacing.ts) are used for these kind of placements as it in general results in a more consistent view.

What is more important is that we cannot use pixel values in react native, this will most likely break if we want to build the application for iOS / Android and will only work in the web build.
> All dimensions in React Native are unitless, and represent density-independent pixels.
https://reactnative.dev/docs/height-and-width